src/web/app/pages-instructor/instructor-home-page/__snapshots__/instructor-home-page.component.spec.ts.snap

```diff
@@ -0,0 +1,1086 @@
+                <thead>
+                  <tr
+                    ng-reflect-ng-class="[object Object]"
+                  >
+                    
+                    <th
+                      class="sortable-header"
+                    >
+                       Session Name 
+                      
+                      <span
+                        class="text-secondary"
+                        ng-reflect-ng-class="[object Object]"
+                      >
+                        <i
+                          class="fas fa-sort"
+                        />
+                      </span>
+                      
+                      
+                    </th>
+                    
+                    <th
+                      class="sortable-header"
+                    >
+                       Start Date 
+                      
+                      <span
+                        class="text-secondary"
+                        ng-reflect-ng-class="[object Object]"
+                      >
+                        <i
+                          class="fas fa-sort"
+                        />
+                      </span>
+                      
+                      
+                    </th>
+                    
+                    <th
+                      class="sortable-header"
+                    >
+                       End Date 
+                      
+                      <span
+                        class="text-secondary"
+                        ng-reflect-ng-class="[object Object]"
+                      >
+                        <i
+                          class="fas fa-sort"
+                        />
+                      </span>
+                      
+                      
+                    </th>
+                    <th>
+                      Submissions
+                    </th>
+                    <th>
+                      Responses
+                    </th>
+                    <th
+                      container="body"
+                      ng-reflect-container="body"
+                      ng-reflect-ngb-tooltip="Number of students submitted /"
+                      ngbtooltip="Number of students submitted / Class size"
+                    >
+                      <span
+                        class="tool-tip-decorate"
+                      >
+                        Response Rate
+                      </span>
+                    </th>
+                    <th>
+                      Action(s)
+                    </th>
+                  </tr>
+                </thead>
+                <tbody>
+                  
+                  <tr>
+                    
+                    <td>
+                      Feedback 1
+                    </td>
+                    
+                    <td
+                      container="body"
+                      ng-reflect-container="body"
+                      ng-reflect-ngb-tooltip="Thu, 01 Jan 1970, 07:30 AM +07"
```

Comment:
Might want to change the values in the test case to see if these are being rendered correctly too.